src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -105,13 +152,12 @@ const Inventory = ({
             description: rule.description,
           },
         ],
-        systems: selected,
+        systems: selectedIds,
       };
     }
   };
 
   const onRemediationCreated = (result) => {
-    onSelectRows(-1, false);
```

Comment:
I believe this was meant to clear the selection after remediation. The bulk select hook returns a `selectNone` function, which we could replace this with.